domains/kory.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Kory Porter's personal website",
+  "repo": "https://github.com/korziee",
+  "owner": {
+    "username": "korziee",
+    "email": ""
```

We need a way to contact you in case we need to inform you of any changes. If you do not wish to share your email address, you can specify your twitter/mastodon handle or any other social media username instead.

```suggestion
  "email": "",
  "twitter": "your-username"
```